sdcm/results_analyze.py

```diff
@@ -58,6 +58,46 @@ def __call__(self, *args, **kwargs):
         return None
 
 
+class PerformanceQueryFilter(QueryFilter):
+    MAIN_JOB_FOLDERS = ["Perf-Regression",
+                        "scylla-master"]
+
+    def filter_test_details(self):
+        test_details = self.build_filter_job_name()
+        test_details += self.test_cmd_details()
+        test_details += ' AND test_details.time_completed: {}'.format(self.date_re)
+
+        test_details += ' AND {}'.format(self.build_filter_test_name())
+        return test_details
+
+    def build_filter_job_name(self):
+        job_name = self.test_doc['_source']['test_details']['job_name'].split("/")
+        if job_name[0] in self.MAIN_JOB_FOLDERS:
+            base_job_name = job_name[1]
+        else:
```

Comment:
Need elsif and make sure it's either perf_regression or scylla-master.
else:
if none of them, we don't compare to them.

Reply:
Is it ok, that if job name starts from perf-regressoin or scylla-master of located in folder perf-regression or scylla-master, then compare results this jobs only otherwise compare explictly by job name (will be good for staging, or future custom performance results)

Reply:
Fixed code to correctly build query for jobs